3-enrich/scala-common-enrich/src/test/scala/com.snowplowanalytics.snowplow.enrich.common/enrichments/registry/apirequest/ApiRequestEnrichmentIntegrationTest.scala

```diff
@@ -309,4 +474,41 @@ class ApiRequestEnrichmentIntegrationTest extends Specification with ValidationM
       }
     }
   }
+
+  def e3 = {
+    val config  = ApiRequestEnrichmentConfig.parse(IntegrationTests.configuration3, SCHEMA_KEY)
```

Comment:
I think we just should put the `configuration` into this spec, rather than referring to them as to `1`/`2`/`3`